src/main/java/ru/spbstu/repository/AnnotationRepository.java

```diff
@@ -0,0 +1,12 @@
+package ru.spbstu.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import ru.spbstu.model.Annotation;
+
+import java.util.List;
+
+@Repository
+public interface AnnotationRepository extends CrudRepository<Annotation, Long> {
+  List<Annotation> findAllByVariant_Id(Long id);
```

Comment:
Let's introduce new method that finds all annotations by list of variant codes.

Reply:
+